src/modules/entities/modules/reports/services/sagas/fetchReports.js

```diff
@@ -17,7 +17,9 @@ function* fetchReports() {
         const results = [];
 
         snapshot.forEach(documentSnapshot => {
-            results.push(storage.ref(documentSnapshot.id).getDownloadURL());
+            results.push(storage.ref('6HyDpbhbfOqNxxvzTXb8.png').getDownloadURL());
+            // TO WORK with official firebase
```

Comment:
always write // TODO - when you wanna add todo comment, for better searching through the app

Reply:
ok, renamed it